src/hooks/mongodb.test.js

```diff
@@ -1,11 +1,5 @@
 import { HandlerInputsBuilder } from '../../testUtils/handlerInputsBuilder';
-import {
-  getMockedMongoClient,
-  promisifyMongoFunc,
-  wrapMongoCollection,
-} from '../../testUtils/mongoMocker';
-import { MongoSpanBuilder } from '../../testUtils/mongoSpanBuilder';
-import { SpansContainer, TracerGlobals } from '../globals';
+import { TracerGlobals } from '../globals';
```

Comment:
note for reviewer: client-specific tests for 3.x and 4.x clients moved to their own files